readme.md

```diff
@@ -1,14 +1,24 @@
-# Ruby Enumerables: Learning Objectives
+# Ruby Enumerables
+
+## Learning Objectives
 
 - Review Ruby arrays and hashes
 - Use Ruby loops to iterate over code blocks.
 - Define what a Ruby enumerable method is.
-- Identify useful Ruby enumerables, including `.each`, `.map` and `.select`.
 - Use enumerables to traverse, sort and modify collections.
+- Identify useful Ruby enumerables, including `.each`, `.map` and `.select`.
+
+## Intro (10 / 10)
+
+Today we will be going over enumerables in Ruby, one of the most powerful modules that comes included in the language out of the box.
 
-## Review: Ruby Collections (15 minutes)
+We will get into how to use enumerables later on this lesson, but first just know that enumerables essentially give us fun ways to play with data in Ruby.
```

Comment:
I feel like 'fun' & 'play' don't capture the nature of enumerables here. I mean, they are fun, but to me, it's really a more readable, expressive way to work with collections of data.